### ISSUE TITLE
MPS Add On Installation error

### ISSUE THREAD

**Issue by KjellRSeeq** [seeq12/seeq-mps#1]
Tried to install MPS from https://seeq12.github.io/seeq-mps/installation.html#user-installation-seeq-data-lab following the instructions provided. 
Step 2 executes as expected.
Step 3 I encounter an error (I tried a few variations, to no positive effect - it seems unable to locate seeq.addons.mps ). Screen clip below:
![image-2021-10-29-16-32-52-831](https://user-images.githubusercontent.com/92815457/140333623-d77d59f4-dc0a-4c42-b44e-6ba06b0b6421.png)

The server in question is running Seeq version R52.1.4 and SPy version 52.1.4.179.18. 

I did also try `python -m seeq.addons.mps` as a command, with the following message:
![image](https://user-images.githubusercontent.com/92815457/140333645-03035e3d-b69a-4099-b3b6-2087465abcbd.png)

I will also note:

- I am an Admin on this server
- We are still running the External Tools notebook on this server, instead of the newer Add Ons notebook

**Comment by jameshiggie** [comment]
Hi KjellRSeeq,

This seems to be an issue with the older version of SPy you are using, please upgrade your version with `pip install seeq==52.3.2.182.25 -U` and try again.

**Comment by KjellRSeeq** [comment]
Thanks James,
I did that install, and it did get through the 2 installation steps for me, until I ran into this error:
![image](https://user-images.githubusercontent.com/92815457/140594568-228cc5c8-7df1-41f5-98bd-ed2c58eab407.png)

Looks like perhaps since in my server version the addon naming is not recognized. I will note that after the step above, I was able to see the notebook in my SDL project, and used that to load it into my External Tool manager manually, which worked. The application is now working and launching from Workbench as expected.
Thanks!
